sdk/src/wallet/operations/participation/event.rs

```diff
@@ -60,12 +61,11 @@ where
                 data: event_data,
                 nodes: vec![options.node.clone()],
             };
-            let account_index = self.details().await.index;
-            self.wallet
+            self.inner
```

Comment:
You sometimes use `inner.storage_manager` and sometimes just `.storage_manager`, harmonise pls

Reply:
This hasn't been addressed

Reply:
Done

Reply:
There is still one `inner`